libs/deps/offer-selector-tool.js

```diff
@@ -0,0 +1,3 @@
+/*! For license information please see index.js.LICENSE.txt */
```

Comment:
If a new dev has to update this file with the newest version of the OST is there documentation that shows them how to do it? I think even adding a small page on the milo wiki with a bit of documentation would help.

Reply:
Done. Added documentation in OST block: [https://github.com/adobecom/milo/blob/20427427905c0b38d868099625de68b45772f700/libs/blocks/ost/README.md](https://github.com/adobecom/milo/blob/20427427905c0b38d868099625de68b45772f700/libs/blocks/ost/README.md) ✅